db/migrate/20220523152453_create_bookings.rb

```diff
@@ -1,11 +1,10 @@
 class CreateBookings < ActiveRecord::Migration[6.1]
   def change
     create_table :bookings do |t|
-      t.string :status
+      t.string :status, default: true
```

Comment:
what is the definition of this status attribute?

Reply:
This enables us to pass this the status variable as True, once booking is complete.  Ensures no need to pre define status values at this point.